src/components/AddTaskCard/AddTaskCard.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import { sanitizeText } from "src/utils/sanitizeText";
+import "./AddTaskCard.scss";
+
+function AddTaskCard({ onCreateTask }) {
+  const [taskTitle, setTaskTitle] = useState("");
+  const sanitizedTaskTitle = sanitizeText(taskTitle);
+  const isTaskButtonDisabled = sanitizedTaskTitle === "";
+
+  function handleChange(event) {
+    setTaskTitle(event.target.value);
+  }
+
+  function handleAddTask() {
+    onCreateTask(sanitizedTaskTitle);
+  }
+
+  return (
+    <>
+      <textarea
+        className="task__textarea"
+        autoFocus
+        value={taskTitle}
+        onChange={handleChange}
+      />
+
+      <button
+        className={`task__save-button ${
+          isTaskButtonDisabled ? "task__save-button-disabled " : ""
+        }`}
```

Comment:
can we use [classnames](https://www.npmjs.com/package/classnames)?

Reply:
@ahr-wtag doesn't it add extra complexity for adding a simple class name?